lib/const.rb

```diff
@@ -102,3 +102,6 @@
 PY3_SETUP_BUILD_OPTIONS = "--executable=#{CREW_PREFIX}/bin/python3"
 PY2_SETUP_BUILD_OPTIONS = "--executable=#{CREW_PREFIX}/bin/python2"
 PY_SETUP_INSTALL_OPTIONS = "--root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX} -O2 --compile --single-version-externally-managed"
+
+CREW_FIRST_PACKAGES = %w[curl git pixz shared_mime_info]
+CREW_LAST_PACKAGES = %w[ghc mandb gtk3 gtk4 sommelier]
```

Comment:
Since these packages are now assigned to a constant, can we use the constant in the crew function `def resolve_dependencies` also?
```
  first_packages = %w[curl git pixz shared_mime_info]
  last_packages = %w[ghc mandb gtk3 gtk4 sommelier]
```
The above lines can be replaced by the constants instead.

Reply:
Yes absolutely.

Reply:
But I'm not actually sure the resolve_dependencies code for reordering to the front of the queue works. Shouldn't it be invoked during upgrades? It doesn't appear to be ...

Reply:
@uberhacker Made some changes here. See if it looks reasonable?

Reply:
(Still not sure if this is actually invoked...)